## Multimodal, multidataset etc.
Sometimes translation is mixed with integration and the papers are not repeated.
- [Multimodal attention-based variational autoencoder for clinical risk prediction](https://ieeexplore.ieee.org/document/10385792/metrics#metrics) 
- +++ [Robust single-cell matching and multimodal analysis using shared and distinct features](https://www.nature.com/articles/s41592-022-01709-7)
- ++++[Biologically informed deep learning to query gene programs in single-cell atlases](https://www.nature.com/articles/s41556-022-01072-x)
- ## Translation
	- +++[scButterfly: a versatile single-cell cross-modality translation method via dual-aligned variational autoencoders](https://www.nature.com/articles/s41467-024-47418-x?fromPaywallRec=false)
	- [Multi-Domain Translation by Learning Uncoupled Autoencoders](https://arxiv.org/abs/1902.03515)
	- paywall [Recovery of missing single-cell RNA-sequencing data with optimized transcriptomic references](https://www.nature.com/articles/s41592-023-02003-w?fromPaywallRec=false)
- ## Domain adaptation
	- +++[Domain-Specific Batch Normalization for Unsupervised Domain Adaptation](https://arxiv.org/pdf/1906.03950)
		- seems not too hard to add it to the current models
		- could be integrated actually with the batch conditioning model which would provide these parameters. but it would not be as straightforward as using conditioning, but would perhaps allow to adjust the parameters better in case of small number of samples in a given dataset
		- how it works when we only fine tune in this way?
- ## Integration - mainly single cell data
	-> omivae [Integrated Multi-omics Analysis Using Variational Autoencoders: Application to Pan-cancer Classification](https://arxiv.org/abs/1908.06278) | babel [BABEL enables cross-modality translation between multiomic profiles at single-cell resolution](https://pubmed.ncbi.nlm.nih.gov/33827925/) | aae [Multi-domain translation between single-cell imaging and sequencing data using autoencoders](https://www.nature.com/articles/s41467-020-20249-2) note [[OmiVAE, BABEL, AAE GAN latent matching]]
	-->[A unified computational framework for single-cell data integration with optimal transport](https://www.nature.com/articles/s41467-022-35094-8?fromPaywallRec=false) note [[A uniﬁed computational framework for single-cell data integration with optimal transport]]
	- ++[Semi-supervised integration of single-cell transcriptomics data](https://www.nature.com/articles/s41467-024-45240-z?fromPaywallRec=false)
	- ++[MAGAN: Aligning Biological Manifolds](https://arxiv.org/abs/1803.00385)
	- ++ [scMoMaT jointly performs single cell mosaic integration and multi-modal bio-marker detection][https://www.nature.com/articles/s41467-023-36066-2?fromPaywallRec=false]
	- ++[Multi-omics single-cell data integration and regulatory inference with graph-linked embedding](https://www.nature.com/articles/s41587-022-01284-4?fromPaywallRec=false)
	- [Probabilistic embedding, clustering, and alignment for integrating spatial transcriptomics data with PRECAST](https://www.nature.com/articles/s41467-023-35947-w?fromPaywallRec=false)
	- [Integration of spatial and single-cell data across modalities with weakly linked features](https://www.nature.com/articles/s41587-023-01935-0?fromPaywallRec=false)
	- + [Paired single-cell multi-omics data integration with Mowgli](https://www.nature.com/articles/s41467-023-43019-2?fromPaywallRec=false)
	- [Scanorama: integrating large and diverse single-cell transcriptomic datasets](https://www.nature.com/articles/s41596-024-00991-3?fromPaywallRec=true)
	- [Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space](https://www.nature.com/articles/s41467-022-33758-z)
	- [scCobra: Contrastive cell embedding learning with domain-adaptation for single-cell data integration and harmonization](https://www.biorxiv.org/content/10.1101/2022.10.23.513389v2)
---
### Transformers
- +++++ [REPO WITH SC TRANSFORMER PAPERS](https://github.com/theislab/single-cell-transformer-papers)
->  [scGPT: Towards Building a Foundation Model for Single-Cell Multi-omics Using Generative AI](https://www.biorxiv.org/content/10.1101/2023.04.30.538439v1.full) note [[scGPT toward building a foundation model for single-cell multi-omics using generative AI]]
- +++++ [scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data](https://www.nature.com/articles/s42256-022-00534-z)
- +++++ [scMulan: A Multitask Generative Pre-Trained Language Model for Single-Cell Analysis](https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_57)
- +++++ [Transfer learning enables predictions in network biology](https://www.nature.com/articles/s41586-023-06139-9)
- +++++ [CellPLM: Pre-training of Cell Language Model Beyond Single Cells](https://openreview.net/forum?id=BKXvPDekud)
- +++++ [Large-scale foundation model on single-cell transcriptomics](https://www.nature.com/articles/s41592-024-02305-7)
- [scmFormer Integrates Large‐Scale Single‐Cell Proteomics and Transcriptomics Data by Multi‐Task Transformer](https://www.researchgate.net/publication/378963194_scmFormer_Integrates_Large-Scale_Single-Cell_Proteomics_and_Transcriptomics_Data_by_Multi-Task_Transformer)
- [Single-Cell Multimodal Prediction via Transformers](https://arxiv.org/pdf/2303.00233)
- [scGREAT: Transformer-based deep-language model for gene regulatory network inference from single-cell transcriptomics](https://www.sciencedirect.com/science/article/pii/S258900422400573X)
- transformer that Marcin mentioned [LORA: LOW-RANK ADAPTATION OF LARGE LANGUAGE MODELS](https://arxiv.org/pdf/2106.09685)
### Diffusion
- [scRDiT: Generating single-cell RNA-seq data by diffusion transformers and accelerating sampling](https://arxiv.org/html/2404.06153v1)